lib/transactions/crypto.js

```diff
@@ -71,22 +69,22 @@ function getBytes(transaction, skipSignature, skipSecondSignature) {
 
 	}
 
-	var bb = new ByteBuffer(1 + 4 + 32 + 8 + 8 + 21 + 64 + 64 + 64 + assetSize, true);
+	bb = new ByteBuffer(1 + 4 + 32 + 8 + 8 + 21 + 64 + 64 + 64 + assetSize, true);
 	bb.writeByte(transaction.type);
 	bb.writeInt(transaction.timestamp);
 
 	var senderPublicKeyBuffer = new Buffer(transaction.senderPublicKey, "hex");
-	for (var i = 0; i < senderPublicKeyBuffer.length; i++) {
+	for (i = 0; i < senderPublicKeyBuffer.length; i++) {
```

Comment:
My preference would lean towards using let in each of the for loops versus defining it at the top using var

Reply:
Would need to go into a different PR.